### PR TITLE
S1 t4 connect and read from database

### PR DESCRIPTION
**Pull request by robin-hodge**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1,30 +1,22 @@
 <?php
+require 'src/functions.php';
+
+$dbString = 'dreams';
```
Comment (chloe-smart) [inline index.php]: is this line of code necessary? as discussed:)

File: src/functions.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    $query = $db->prepare('SELECT * FROM dreams;');
+    $query->execute();
+    $dreams =  $query->fetchALL();
+    return $dreams;
+}
+
+function displayDreamDetails(array $dreams): string{
+    $dreamOutput ='';
+    foreach($dreams as $dream){
+        $dreamOutput .="<h3>" . $dream['dream_title'] . "</h3>
```
Comment (chloe-smart) [inline src/functions.php:6]: what does the dreamOutput .= do? just haven't seen this before so intrigued

**Review by chloe-smart — COMMENTED** [review]
Could maybe have a folder db.php where you connect to the db for extra clarity

File: index.php (hunk 1 above)
Comment (robin-hodge) [reply index.php]: I will remove it!

File: src/functions.php (hunk 2 above)
Comment (robin-hodge) [reply src/functions.php:6]: left, as discussed!

**Review by chloe-smart — APPROVED** [review]
Great work:)

**Pull request merged**